go/op_exporter/main.go

```diff
@@ -86,13 +98,53 @@ func main() {
 	})
 	go getRollupGasPrices()
 	go getBlockNumber(&health)
+	if *enableK8sQuery {
+		client, err := k8sClient.Newk8sClient()
+		if err != nil {
+			log.Fatal(err)
+		}
+		go getSequencerVersion(&health, client)
+	}
+	//go getAllowedMethods(&health, client)
 	log.Infoln("Listening on", *listenAddress)
 	if err := http.ListenAndServe(*listenAddress, nil); err != nil {
 		log.Fatal(err)
 	}
 
 }
 
+func getSequencerVersion(health *healthCheck, client *kubernetes.Clientset) {
+	ns, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
+	if err != nil {
+		log.Fatalf("Unable to read namespace file: %s", err)
+	}
+	for {
```

Comment:
This could also be turned into a `range` over a `time.Ticker` if you end up implementing https://github.com/ethereum-optimism/optimism/pull/1286/files#r677776976 which would remove the need to have 2 sleeps

Reply:
Is a ticker preferred over sleeps?

Reply:
Kept this ticker local to the k8s loop as the metric collection loop can/should have a different interval.